packages/ui-extensions/src/surfaces/customer-account/components/CustomerAccountAction.d.ts

```diff
@@ -1,3 +1,5 @@
+import {BaseElementPropsWithChildren} from './shared';
```

Comment:
would be nice to use a shared version with checkout once they have updated theirs here as well.